CHANGES

```diff
@@ -7,8 +7,16 @@ Note that ``RB_ID=#`` correspond to associated messages in commits.
 6.x
 -----
 
+6.34.0
```

Comment:
leave this next version out please. our release tooling automatically bumps the 6.x to the next version when its release time.